nacho_api/settings.py

```diff
@@ -99,6 +99,7 @@
 )
 
 # Parse database configuration from $DATABASE_URL
+# note: this setting must be commented out to run locally.
```

Comment:
Good catch. I wonder if we could automate this later -- just set some flag somewhere that says if we're running locally or not, and then things like this get automatically set.